src/transformers/modeling_fsmt.py

```diff
@@ -1003,6 +1001,14 @@ def set_output_embeddings(self, value):
 )
 class FSMTForConditionalGeneration(PretrainedFSMTModel):
     base_model_prefix = "model"
+    authorized_missing_keys = [
+        "model.encoder.embed_positions.weight",
+        "model.decoder.embed_positions.weight",
+    ]
+    state_dict_no_save_keys = [
```

Comment:
`keys_to_never_save` ?

Reply:
I'm up for any reasonable name, but let's have those two be similar naming-wise:
```
    authorized_missing_keys = [
        "model.encoder.embed_positions.weight",
        "model.decoder.embed_positions.weight",
    ]
    state_dict_no_save_keys = [
        "model.encoder.embed_positions.weight",
        "model.decoder.embed_positions.weight",
    ]
```

does it make sense? - i.e. `_keys` last/first?

Reply:
So perhaps `authorized_never_save_keys`?

Reply:
or to match your suggestion and then rewrite the existing one?
```
keys_missing_ok
keys_to_never_save
```

Reply:
ok, to move this along, I used your suggestion and will discuss this in a separate issue.